tools/registry.py

```diff
@@ -264,20 +268,35 @@ def add(self, module, override=False):
     if module.presubmit_yml:
       shutil.copy(module.presubmit_yml, presubmit_yml)
     else:
-      platforms = {
-          "centos7": {},
-          "debian10": {},
-          "ubuntu2004": {},
-          "macos": {},
-          "windows": {},
-      }
-      for _, config in platforms.items():
-        if module.build_targets:
-          config["build_targets"] = module.build_targets.copy()
-        if module.test_targets:
-          config["test_targets"] = module.test_targets.copy()
+      PLATFORMS = ["centos7", "debian10", "ubuntu2004", "macos", "windows"]
+      presubmit = {}
```

Comment:
nit: can we use dict literals for this, and lines 276, 284, 289, etc.?

Reply:
Done